api-docs/swagger-v1.1.yml

```diff
@@ -598,7 +600,7 @@ paths:
                 $ref: '#/definitions/total'
   /products/usages:
     get:
-      summary: list usages (categories) for a product query
+      summary: list usages (categories) for a product query, implicit usages included
```

Comment:
> Ik vind dit eigenlijk voor zich spreken (zeker als je de output een keer bekijkt).

Dat ben ik met je eens. Maar ik dacht dat dit is wat je [hiermee](https://github.com/q-m/questionmark-api/pull/2203#issuecomment-284386328) bedoelde.